client/src/views/Landing.jsx

```diff
@@ -9,12 +9,13 @@ import { ColorModeContext, tokens } from "../theme";
 import Hero from "../components/landing-components/Hero";
 
 const Landing = () => {
+  const [listingList, setListingList] = useState([]);
```

Comment:
more formatting.  This is most likely line deletions and spaces, but double check that it's not something actually changing.

Reply:
dawdawdawd